Makefile

```diff
@@ -148,6 +148,9 @@ distclean:
 install: checkprefix nvim
 	+$(BUILD_TOOL) -C build install
 
+clang-tidy: generated-sources
+	+clang-tidy --config-file .clang-tidy -p build $(FILE)
```

Comment:
We should define a default value for `FILE`, even if it's empty.

Reply:
After some testing I figured it's not worth it, since clang-tidy will not run in parallel on its own. We can iterate on this further later on.